vault/package/init.sls

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% from "vault/map.jinja" import vault with context %}
+
+include:
+  - .install
+{%- if vault.verify_download %}
+  - .gpg
+  - .signature
+{%- endif %}
```

Comment:
Cosmetic issue which can be ignored: its useful to indent Jinja blocks the same as the surrounding code, i.e.:

```sls
  {%- if vault.verify_download %}
  - .gpg
  - .signature
  {%- endif %}
```